OS-9 Patch

jimrye opened the pull request:
Patch to detect when OS-9 is running and change to the start of screen memory for text copy. Works in OS-9 text mode only.

jimrye committed:
Merge pull request #3 from snowdogz/VCCE-VCC_2.0.1c

Vcce vcc 2.0.1c

jimrye committed:
Merge pull request #6 from VCCE/VCC_2.0.1c

Grabbing the latest from the main repo

jimrye committed:
OS-9 Patch

OS-9 Patch

jimrye committed:
Merge branch 'copy-paste' of https://github.com/snowdogz/VCC into copy-paste

BGPierce commented:
I'll check it out within the next couple of days and let you know how it works.
Bill P.


-----Original Message-----
From: JimRye <notifications@github.com>
To: VCCE/VCC <VCC@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Sent: Tue, Nov 10, 2020 8:40 pm
Subject: [VCCE/VCC] OS-9 Patch (#22)

Patch to detect when OS-9 is running and change to the start of screen memory for text copy. Works in OS-9 text mode only.
You can view, comment on, or merge this pull request online at:
  https://github.com/VCCE/VCC/pull/22
Commit Summary
   
   - Merge pull request #3 from snowdogz/VCCE-VCC_2.0.1c
   - Merge pull request #6 from VCCE/VCC_2.0.1c
   - OS-9 Patch
   - Merge branch 'copy-paste' of https://github.com/snowdogz/VCC into copy-paste

File Changes
   
   -  M coco3.cpp (44) 
   -  M tcc1014mmu.c (4) 
   -  M tcc1014mmu.h (1) 

Patch Links:
   
   - https://github.com/VCCE/VCC/pull/22.patch
   - https://github.com/VCCE/VCC/pull/22.diff
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.

jimrye committed:
Improved Copy Text

Copy Text uses the active text screen. Needs testing on OS9 and Nitros9. Added error message when attempting to copy from graphics modes. Added a warning when user attempts to paste into graphics mode.
